python/pip_install/requirements.bzl

```diff
@@ -29,7 +31,7 @@ def compile_pip_requirements(
         requirements_linux = None,
         requirements_windows = None,
         visibility = ["//visibility:private"],
-        tags = None,
+        tags = _DEFAULT_TAGS,
```

Comment:
Adding defaults makes sense if we always need defaults, but I wonder if we always do. I would like to see documentation on our defaults and how to override them.  Is there a creative way to do this without adding defaults?